Kristian-Repizo/tarea5/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Puzzle Game</title>
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+
+<header>
+    <h1>PUZZLE GAME</h1>
+</header>
+<body>
+    <article id="container-puzzle">
+        <section class="numbers">13</section>
+        <section class="numbers">10</section>
+        <section class="numbers">2</section>
+        <section class="numbers">3</section>
+        <section class="numbers">9</section>
+        <section class="numbers">12</section>
+        <section class="numbers">15</section>
+        <section class="numbers">4</section>
+        <section class="numbers">7</section>
+        <section class="numbers">11</section>
+        <section class="numbers">1</section>
+        <section class="numbers">8</section>
+        <section class="numbers">6</section>
+        <section class="numbers">14</section>
+        <section class="numbers">5</section>
+        <section class="numbers"></section>
+    </article>
+</body>
+<script src="/js/main.js"></script>
```

Comment:
@Kristian-Repizo , you should use relative paths

`<script src="js/main.js"></script>`